mlrun/run.py

```diff
@@ -920,13 +925,16 @@ def wait_for_pipeline_completion(
     :param expected_statuses:  list of expected statuses, one of [ Succeeded | Failed | Skipped | Error ], by default
                                [ Succeeded ]
     :param namespace:  k8s namespace if not default
+    :param remote:     read kfp data from mlrun service (default=True)
 
     :return: kfp run dict
     """
     if expected_statuses is None:
         expected_statuses = [RunStatuses.succeeded]
     namespace = namespace or mlconf.namespace
-    remote = not get_k8s_helper(silent=True).is_running_inside_kubernetes_cluster()
+    remote = (
+        remote or not get_k8s_helper(silent=True).is_running_inside_kubernetes_cluster()
+    )
```

Comment:
If someone will give `remote=False` and they are not inside k8s cluster, it will still read from remote, which doesn't makes sense
I think we can just remove these 3 lines, and simply use the remote arg (which by default will be `True`)

Reply:
ok